sqlmesh/core/context.py

```diff
@@ -2243,9 +2255,9 @@ def _register_notification_targets(self) -> None:
 
     def _load_materializations_and_signals(self) -> None:
         if not self._loaded:
-            for context_loader in self._loaders.values():
-                with sys_path(*context_loader.configs):
-                    context_loader.loader.load_materializations(self)
+            for loader in self._loaders:
+                with sys_path(loader.context_path):
+                    loader._load_materializations(self)
```

Comment:
Let's not call private methods

Reply:
Also you pass `context` as the first argument which is no longer in the function signature. I hope we have tests that cover this